src/Text/Pandoc/Readers/TikiWiki.hs

```diff
@@ -344,9 +344,14 @@ listItemLine nest = lineContent >>= parseContent
       return $ filterSpaces content ++ "\n" ++ Data.Maybe.fromMaybe "" continuation
     filterSpaces = reverse . dropWhile (== ' ') . reverse
     listContinuation = string (replicate nest '+') >> lineContent
+    removeTrailingSpace []     = []
+    removeTrailingSpace (x:[]) = if x == B.space
```

Comment:
Assuming this isn't replaced with `dropWhileEnd`, this seems unnecessarily wordy.  Why not
```
removeTrailingSpace [B.space] = ...
```